modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1251,6 +1284,24 @@ CV__DNN_INLINE_NS_BEGIN
     /** @overload */
     CV_EXPORTS_W void blobFromImagesWithParams(InputArrayOfArrays images, OutputArray blob, const Image2BlobParams& param = Image2BlobParams());
 
+    /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+     *  @param rBlob rect in blob coordinates.
+     *  @param sizeOri original input image size inserted in blob.
+     *  @param size spatial size for output image
+     *  @param crop flag use in blobFromImage called
+     *  @returns rectangle in original image coordinates.
+     */
+    CV_EXPORTS_W Rect blobRectToImageRect(Rect rBlob, Size sizeOri, const Size& size, bool crop);
+
+    /** @brief Get rectangle coordinates in original image system from rectangle in blob coordinates.
+     *  @param rBlob rect in blob coordinates.
+     *  @param sizeOri original input image size inserted in blob.
+     *  @param size spatial size for output image
+     *  @param crop flag use in blobFromImage called
+     *  @returns rectangle in original image coordinates.
+     */
+    CV_EXPORTS_W Rect2d blobRectToImageRect(Rect2d rBlob, Size sizeOri, const Size& size, bool crop);
```

Comment:
@LaurentBerger, just to verify that these methods have not been missed during transition under `Image2BlobParams`

Reply:
@LaurentBerger, this is still actual. Do we want these two methods to be outside of the `Image2BlobParams`?

Reply:
If the problem in the `crop`, shall we add a new filed `bool crop` to `Image2BlobParams` instead?

Reply:
crop is paddingmode in Image2BlobParams https://github.com/opencv/opencv/blob/4.x/modules/dnn/src/dnn_utils.cpp#L108-L109

Reply:
> this is still actual. Do we want these two methods to be outside of the `Image2BlobParams`?
Actually you can get a blob using [blobFromImage](https://docs.opencv.org/4.x/d6/d0f/group__dnn.html#ga98113a886b1d1fe0b38a8eef39ffaaa0) or [blobFromImageWithParams](https://docs.opencv.org/4.x/d6/d0f/group__dnn.html#gadc12e5f4a801fd3c1d802f4c8c5d311c)
Then I use same idea to get rect  without param or with param

Reply:
I see. If there is no objections, I'd like recommend to keep only ones at params